src/main/java/driver/SingletonDriver.java

```diff
@@ -1,5 +1,31 @@
 package driver;
 
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import java.util.concurrent.TimeUnit;
+
 public class SingletonDriver {
+    private static WebDriver driver;
+    private static final String DRIVER_PATH  = "src/main/resources/chromedriver.exe";
+    public static final String THE_URL = "https://www.bookdepository.com";
```

Comment:
seems that this url is not used (or can be moved to the Constants class)

Reply:
done